src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(./images/star.svg) ;
+  background-repeat: no-repeat;
+  background-position: center;
+  color: #e5e5e5;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+  color: #ffde6a;
```

Comment:
The 'color' property is not required for the active stars as they are background images, remove this line.